oauthproxy.go

```diff
@@ -379,13 +383,22 @@ func (p *OAuthProxy) ErrorPage(rw http.ResponseWriter, code int, title string, m
 		Message:     message,
 		ProxyPrefix: p.ProxyPrefix,
 	}
-	p.templates.ExecuteTemplate(rw, "error.html", t)
+	err := p.templates.ExecuteTemplate(rw, "error.html", t)
+	if err != nil {
+		logger.Printf("Error rendering error.html template: %s", err.Error())
```

Comment:
I just copied a lot of the existing calls to align styles... do I actually need `err.Error()` or is `err` itself fine with these logger format strings.

Reply:
You don't need `err.Error()`, just `err` is fine. There's also a go tendency to stick with `%v` for `err` objects.
I'd say change them as you touch them to fit with this idiom if that's ok? The below is how it "should" be
```suggestion
		logger.Printf("Error rendering error.html template: %v", err)
```